### PR TITLE
Add frequency bins to message sent to RFI receiver script

### PR DESCRIPTION
**Pull request by james-s-willis**
Solves #748.

### CODE REVIEW

File: python/scripts/rfi_receiver/rfi_receiver.py [hunk 1]
```diff
@@ -486,7 +489,9 @@ def bad_input_listener(thread_id):
     RFIHeaderSize = app.config["chime_rfi_header_size"]
     mode = app.mode
     firstPacket = True
-    PacketSize = RFIHeaderSize + 4 * local_freq * num_elements
```
Comment (james-s-willis) [inline python/scripts/rfi_receiver/rfi_receiver.py:489]: @andrerenard I think this line was originally wrong, as the `fault_counter` being sent by `kotekan` in `rfiBadInputFinder.cpp` is a byte array not an integer array. So it should be:
`PacketSize = RFIHeaderSize + local_freq * num_elements`

What do you think?

Comment (andrerenard) [reply python/scripts/rfi_receiver/rfi_receiver.py:489]: I agree.  This looks like it's incorrect in the original code.

File: python/scripts/rfi_receiver/rfi_receiver.py [hunk 2]
```diff
@@ -332,7 +327,10 @@ def data_listener(thread_id):
     mode = app.mode
     firstPacket = True
     vdifPacketSize = global_freq * 4 + RFIHeaderSize
-    chimePacketSize = RFIHeaderSize + 4 * local_freq
+    # CHIME packet: RFIHeader + frequency_bins[local_freq] + rfi_avg[local_freq]
+    chimePacketSize = RFIHeaderSize + 4 * local_freq + 4 * local_freq
```
Comment (andrerenard) [inline python/scripts/rfi_receiver/rfi_receiver.py:331]: Currently the size and shape of the packet both depend on both sizes having the same config.  Could we add some checks on this side to make sure the contents of the header match what this config is set to (e.g. for local_freq, etc?

Comment (james-s-willis) [reply python/scripts/rfi_receiver/rfi_receiver.py:331]: So there already exists a `HeaderCheck` function on line 253.

Comment (james-s-willis) [reply python/scripts/rfi_receiver/rfi_receiver.py:331]: Is that what you mean?

**Comment by james-s-willis** [comment]
Thanks for reviewing. What's the best way to test this?

File: python/scripts/rfi_receiver/rfi_receiver.py (hunk 2 above)
Comment (andrerenard) [reply python/scripts/rfi_receiver/rfi_receiver.py:331]: Ah perfect!

**Pull request merged**